### DISCUSSION TITLE
I can't use this program

### DISCUSSION THREAD

**Discussion by jk9746987** [fraxiinus/ReplayBook#221] [Troubleshooting]
Hello, when I update LOL version,
I can't use replay
My program picture's character, All of the exclamation mark,
And when I use replay function, My program always loading, and can't enter the replay
The screen is black
How can I resolve this problem?

**Comment by fraxiinus** [comment]
To fix the problem with images not loading (exclamation mark) download the images using the menu:
![image](https://user-images.githubusercontent.com/8365233/206814469-0a6c2e0f-768e-4778-bfbf-90add18d1ce7.png)

If stuck loading, check if the "Locale" is set correctly, it must match your league of legends locale:
![image](https://user-images.githubusercontent.com/8365233/206814527-ee481251-04ae-4dd8-9fc8-1e413cff5539.png)

**Reply by jk9746987** [reply]
thanks your help, I resolve all problem.
but...how can I download early LOL version?
replays.xyz can't use now.
Do you know other website let me download early LOL version?

**Reply by fraxiinus** [reply]
Unfortunately replays.xyz was the only one I knew about.
